--- original/program.py
+++ candidate/program.py
@@ -3,19 +3,19 @@
 def load_default(stream, source):
     import configparser
     parser = configparser.ConfigParser()
-    parser.readfp(stream)
+    parser.read_file(stream)
     return parser
 
 
 def load_positional(stream, source):
     from configparser import ConfigParser
     parser = ConfigParser()
-    parser.readfp(stream, source)
+    parser.read_file(stream, source)
     return parser
 
 
 def load_keyword(stream, source):
     import configparser as config
     parser = config.ConfigParser()
-    parser.readfp(fp=stream, filename=source)
+    parser.read_file(f=stream, source=source)
     return parser
